mne_qt_browser/_pg_figure.py

```diff
@@ -417,6 +417,10 @@ def toggle_bad(self, x=None):
 
             # Update line color status
             self.isbad = not self.isbad
+            if self.isbad:
+                self.setZValue(0)
+            else:
+                self.setZValue(1)
```

Comment:
Looks like this will only work with an interactive toggle. Does it already work properly for channels that are in `info['bads']` on initial plot?

Reply:
Arf, we are not running through that method on init as I thought..

Reply:
Next commit does the trick, but it feels a bit ugly.. let me check if I can improve this somehow.

Reply:
Rather than code dup naively I would have expected it to be in `self.update_color()` because it looks like that's called by this method, and I'd assume called in `__init__` as well. Did you think about/consider this?